challenge_two/src/utils/constants.ts

```diff
@@ -0,0 +1,18 @@
+export const V3_FACTORY_ADDRESS: string =
+  "0x1F98431c8aD98523631AE4a59f267346ea31F984";
+
+export const SWAP_EVENT = [
+  "event Swap(address indexed sender, address indexed recipient, int256 amount0, int256 amount1, uint160 sqrtPriceX96, uint128 liquidity, int24 tick)",
+];
+export const TRANSFER_EVENT = [
+  "event Transfer (address indexed from, address indexed to, uint256 value)",
+];
```

Comment:
since this is only for testing purposes, you can create this inside agent.spec.ts